tests/taker_cli.rs

```diff
@@ -0,0 +1,321 @@
+mod test {
```

Comment:
No need for a mod here. Everything is tests folder are considered within the default `test` modules.

Just mark the actual test function with `#[test]` and it should be good.